client/src/components/MatchingApplicantsAndOppotunities.js

```diff
@@ -47,57 +47,81 @@ class MatchingApplicantsAndOppotunities extends Component {
+        return { ...opportunity, percentage };
+      })
+      .sort(this.compare);
 
     return (
-      <div>
-        {this.state.opportunitiesList.map(opportunity => {
-          const matching = opportunity.skills.filter(skill => {
-            const matchingSkills = this.state.applicantSkills.includes(skill);
-            return matchingSkills;
-          }).length;
-          const percentage = (
-            (matching / opportunity.skills.length) *
-            100
-          ).toFixed(2);
-          return (
-            <Card
-              key={opportunity.opportunity_id}
-              as={Link}
-              to={
-                getLoggedInUserData() &&
-                `/opportunities/${opportunity.opportunity_id}`
-              }
-            >
-              <Card.Content>
-                <Card.Header>{opportunity.opportunity_title}</Card.Header>
-                <Card.Description>{opportunity.description}</Card.Description>
-                <CircularProgressbarWithChildren
-                  strokeWidth
-                  value={percentage}
-                  text={`${percentage}%`}
-                  styles={buildStyles({
-                    strokeLinecap: "butt"
-                  })}
+      <Grid stackable>
+        <Grid.Row columns={3}>
+          {matchingOpportunities.map(opportunity => {
+            return (
+              <Grid.Column>
+                {" "}
+                <Card
+                  color={opportunity.percentage > 66 ? "green" : "red"}
+                  key={opportunity.opportunity_id}
+                  as={Link}
+                  to={
+                    getLoggedInUserData() &&
+                    `/opportunities/${opportunity.opportunity_id}`
+                  }
                 >
-                  {percentage > 60 ? (
-                    <img
-                      style={{ width: 50, marginTop: -180 }}
-                      src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRePtynQytlAevO-zkIk1hJHVWGr0LmbNhm7MSbUIz63g3puHya"
-                      alt="cherry"
-                    />
-                  ) : null}
-
-                  <div style={{ fontSize: 12, marginTop: -5 }}></div>
-                </CircularProgressbarWithChildren>
-              </Card.Content>
-            </Card>
-          );
-        })}
-      </div>
+                  <Card.Content>
+                    <Card.Header textAlign="center">
+                      {opportunity.opportunity_title}
+                    </Card.Header>
+                    <Card.Description>
+                      {opportunity.description}
+                    </Card.Description>
+                    <Card.Content textAlign="right" style={{ width: "100px" }}>
+                      <CircularProgressbarWithChildren
+                        value={opportunity.percentage}
+                        text={`${opportunity.percentage}%`}
+                      >
+                        {/* {opportunity.percentage > 60 ? (
```

Comment:
Remove the comment out code if you do not need it.

Reply:
Done